docs/core/versions/selection.md

```diff
@@ -27,10 +27,10 @@ The rest of this document examines those four scenarios.
 
 SDK commands include `dotnet new` and `dotnet run`. The `dotnet` CLI must choose an SDK version for every dotnet command. The .NET Core CLI uses the latest SDK installed on the machine by default, even if:
 
-* The project targets an earlier version.
-* The latest version is a preview version.
+* The project targets an earlier version of the .NET Core runtime.
+* The latest version is a preview version of the .NET Core runtime.
```

Comment:
The text before the list:
>The .NET Core CLI uses the latest SDK installed

So, I think the second item is about the latest .NET Core **SDK** to be a preview, not the runtime. So, the sentence should be:
> The latest version of the .NET Core SDK is a preview version.